packages/keyboard-keys/README.md

```diff
@@ -1,3 +1,45 @@
 # @fluentui/keyboard-keys
 
-Contains a set of keyboard constants for key and keyCode comparison in components
+Contains a set of keyboard constants for key and keyCode comparison in components. This package contains
+**named key values** from [The w3 uievents-key specification](https://www.w3.org/TR/uievents-key/).
+
+Unicode values are not included since there are a lot of locales to consider and they provide no benefit since
+unicode characters can be used directly in code.
+
+# Usage
+
+```ts
+import { Enter } from '@fluentui/keyboard-keys';
+
+const onKeyDown = (e: React.KeyboardEvent) => {
+  if (e.key === Enter) {
+    // ...
+  }
+
+  // Unicode characters 'a', '1', '%'...
+  // should be used directly in code
+  if (e.key === 'a') {
+    // ...
+  }
+};
+```
+
+## Legacy keyCode
+
+In order to migrate easily from `@fluentui/keyboard-key` legacy `keyCode` support is available in this library but
+is not encouraged for reuse since this propoerty has been deprecated for a while and will be removed in future
+standards.
+
+```ts
+import { Enter, a } from '@fluentui/keyboard-keys/lib/keyCodes';
```

Comment:
We should not advertise deep imports, can you please modify this example to avoid them?

Reply:
I know, my original idea was to export `keyCodes` but that will not work thanks to our friend API-Extractor..... any ideas there ?

Reply:
This is the cleanest way I could think of that would not kill bundle-size.

Reply:
@layershifter  after #18920 I was able to remove the deep import